### ISSUE TITLE
C# - Problem returning null shared_pointer.

### ISSUE THREAD

**Issue by Koukan** [scapix-com/scapix#18]
Hi,
First, congratulation for your great job, it's pretty easy and work great.

I have encountered an issue when I return a null shared_ptr. 
I work on a unity plugin, so I use your work to bind my C++ library in C#. And when I return a nullptr, Unity stop working.

A simple example to reproduce the problem:
```
class A : public scapix::bridge::object<A>
{
public:
    static std::shared_ptr<A> Create() { return nullptr; }
};
```

I have worked around the problem for now but I think this is a necessary feature.

**Comment by Boris-Rasin** [comment]
Thank you very much for the bug report.
I have fixed this issue (2610795172aa6afe11ab1168bad706f13134a135) and will make a new release soon.

**Comment by Boris-Rasin** [comment]
I have released Scapix version 1.0.22 with this fix.
Please use latest cmodule version to test it:
https://github.com/scapix-com/cmodule/releases
